wiki/SpriteData.md

```diff
@@ -45,17 +45,115 @@ The `<name>` for this sprite is "effect/blaster impact". The `+` in the file nam
+## Ship Sprite Animation
+Ships can have sprites for several different states of flight: Flying, Firing, Landing, Launching, Jumping, Disabled. Each state can be assigned a sprite similar to the above naming convention, but the tag will be different, as shown below:
+
+```html
+ship <name>
+	sprite <path-to-flying-sprites>
+		...
+	sprite-firing <path-to-firing-sprites>
+		...
+	sprite-landing <path-to-landing-sprites>
+		...
+	sprite-launching <path-to-launching-sprites>
+		...
+	sprite-jumping <path-to-jumping-sprites>
+		...
+	sprite-disabled <path-to-disabled-sprites>
+		...
+```
+
+Note: "sprite" and "sprite-flying" are interchangeable.
+
+Each sprite can be animated individually with the animation parameters as defined below.
+
+
+### Animation Parameters
 
 You can also specify various attributes of the animation. These should be left out if you do not want them:
 
+#### Parameters for all Sprites
+
 * `"frame rate" <fps#>`: frames per second. If no value is provided, a frame rate of 2 will be used.
 * `"frame time" <number#>`: alternative to "frame rate". Specifies the number of game ticks (60 per second) each frame of the animation should last for.
 * `"delay" <number#>`: number of animation frames to delay in between loops of the animation. For example, a four-frame animation with a delay of 4 and a frame rate of 8 FPS will play the animation for half a second, then pause for half a second, then repeat.
 * `"start frame" <number#>`: start at this frame of the animation.
 * `"random start frame"`: start at a random frame of the animation.
 * `"no repeat"`: once the animation has played through once, stay on the last frame until the effect disappears. If this is not defined, the animation loops.
 * `"rewind"`: once the animation has played through to the last frame, play it in reverse. If `"no repeat"` is also defined, the animation will play forward once, then backward once, then stop at the first frame.
+* `"ramp" <up fpsps#> <down fpsps#> `: when the object enters (or exit) a certain state, this allows the frame rate to ramp up (or down) according to the values given in up fpsps (fps per second) and down fpsps.
+* `"random"`: randomize the next frame played in the animation.
+* `"reverse"`: play the animation from the last frame to the first.
+
+#### Parameters for Ship Sprites only
+* `"transition delay" <number#>`: if a ship has requested a transition to a different state, i.e. FIRING to FLYING, wait for a certain number of frames before allowing that transition to occur. This would be used in places where there could be rapid state transitions. For example, if a ship has fired its primary weapon, chances are it might fire its primary weapon again, so instead of immediately transitioning to the flying state, we can wait in the firing state for a certain number of frames before entering the flying state again.
+* `"transition type" <number#>`: currently, three transistion types are supported: "immediate", "finish" and
```

Comment:
```suggestion
* `"transition type" <type>`: currently, three transistion types are supported: "immediate", "finish" and
```
Also, I'm not sure this line break is necessary?

Reply:
Must have snuck its way in as part of the last commit